plugin/inc/menumonthlyplan.class.php

```diff
@@ -0,0 +1,30 @@
+<?php
+class PluginIserviceMenuMonthlyPlan extends CommonGLPI
```

Comment:
I don't think we need a separate class just for the menu info. I would go with a single file for the whole "Monthly Plan" story (and move all the important logic from the monthlyplan.form.php here)
```suggestion
class PluginIserviceMonthlyPlan extends CommonGLPI
```
Do we need to extend the CommonGLPI class? Wouldn't it be enough just to implement the needed functions?

Reply:
Ass discussed:
- Don't remove CommonGLPI extension